android/src/main/java/im/shimo/react/prompt/RNPromptFragment.java

```diff
@@ -198,6 +210,17 @@ public Dialog onCreateDialog(Bundle savedInstanceState) {
         return dialog;
     }
 
+    @Override
+    public void onStart() {
```

Comment:
why not put code after to line 113?

Reply:
I wanted to. I actually tried to put it on line 91 - https://github.com/Noitidart/react-native-prompt-android/blob/dd428a864880209114ff9708f76e21e9d1937aa5/android/src/main/java/im/shimo/react/prompt/RNPromptFragment.java#L91 - where we initially work with postive button. However `d.getButton` returns null until after dialog shows. :(

This person here also encountered same issue - https://tassioauad.com/2016/06/02/dialogfragmentalertdialog-dismiss-automatically-on-click-button/

He says:

> You will need to set this in the onResume() method because getButton() will return null until after the dialog has been shown! This should cause your custom action method to only be called once, and the dialog won’t be dismissed by default.

Reply:
@greedbell @wsong910 would you rather i make the change before accpeting PR? or would you like to find null cases and work around it?